mpas_analysis/ocean/meridional_heat_transport.py

```diff
@@ -0,0 +1,218 @@
+                                             'mpasClimatologySubdirectory')
+    try:
+        os.makedirs(outputDirectory)
+    except OSError:
+        pass
+    outputFileClimo = '{}/meridionalHeatTransport_years{:04d}-{:04d}.nc'.format(
+                       outputDirectory, dictClimo['startYearClimo'],
+                       dictClimo['endYearClimo'])
+    if not os.path.exists(outputFileClimo):
+        print '   Load data...'
+        ds = open_multifile_dataset(
+            fileNames=dictClimo['inputFilesClimo'],
+            calendar=calendar,
+            config=config,
+            simulationStartTime=simulationStartTime,
+            timeVariableName='Time',
+            variableList=variableList,
+            variableMap=variableMap,
+            startDate=dictClimo['startDateClimo'],
+            endDate=dictClimo['endDateClimo'])
+
+        changed, startYear, endYear = \
+            climatology.update_start_end_year(ds, config, calendar)
+        if changed:
+            # update the file name in case the start and end years changed
+            outputFileClimo = \
+                '{}/meridionalHeatTransport_years{:04d}-{:04d}.nc'.format(
+                    outputDirectory, startYear, endYear)
+
+            dictClimo['startYearClimo'] = startYear
+            dictClimo['endYearClimo'] = endYear
+
+        # Compute annual climatology
+        annualClimatology = \
+            climatology.compute_climatology(ds,
+                                            constants.monthDictionary['ANN'],
+                                            calendar)
+
+        # Convert to numpy arrays 
+        # (can result in a memory error for large array size)
+        meridionalHeatTransportLat  = annualClimatology.avgMeridionalHeatTransportLat.values
+        meridionalHeatTransportLatZ = annualClimatology.avgMeridionalHeatTransportLatZ.values
+
+        print '   Creating plots...' 
+        # Define plotting variables
+        mainRunName = config.get('runs', 'mainRunName')
+        #movingAveragePoints = config.getint(sectionName, 'movingAveragePoints')
+        colorbarLabel = '[PW]'
+        xLabel = 'latitude [deg], currently index'
+        yLabel = 'depth [m], currently index'
+    
+        #print '   Plot climatological {} MHT...'.format(region)
+        #title = '{} MHT (ANN, years {:04d}-{:04d})\n {}'.format(
+        #         region, dictClimo['startYearClimo'], dictClimo['endYearClimo'],
+        #         mainRunName)
+        #figureName = '{}/mht{}_{}_years{:04d}-{:04d}.png'.format(
+        #              plotsDirectory, region, mainRunName,
+        #              dictClimo['startYearClimo'], dictClimo['endYearClimo'])
+
+        ########################################################################
+        # Mark P temporary: plot 1D MHT
+        # needs to be replaced with a call to a plotting command in shared.plot.plotting
+        ########################################################################
```

Comment:
@milenaveneziani, is this something you can help with -- getting this code hooked into the shared plotting routines?  Seems pretty simple.  At least I hope so.